xml/System.IO.Enumeration/FileSystemName.xml

```diff
@@ -87,7 +87,17 @@
         <summary> Verifies if the given Win32 expression matches the given name. Supports the following wildcards: '*', '?', '&lt;', '&gt;', '"'. The backslash character '\' escapes.</summary>
```

Comment:
```suggestion
        <summary> Verifies if the given Win32 expression matches the given name. Supports the following wildcards: '*', '?', '&lt;', '&gt;', '"'. The backslash character `\` escapes.</summary>
```

Reply:
This is not a remark, so I don't think the backticks will have an effect. I will only apply the space removal:
```suggestion
         <summary> Verifies if the given Win32 expression matches the given name. Supports the following wildcards: '*', '?', '&lt;', '&gt;', '"'. The backslash character '\' escapes.</summary>
```